protocols/dcutr/Cargo.toml

```diff
@@ -18,7 +18,7 @@ futures = "0.3.1"
 futures-timer = "3.0"
```

Comment:
Same here, you would need to bump the `version` field.

Reply:
thanks, also did it for the remaining crates where the version was not bumped yet.